codebase/config/sync/core.entity_view_display.node.islandora_object.pdfjs.yml

```diff
@@ -60,6 +64,210 @@ targetEntityType: node
 bundle: islandora_object
 mode: pdfjs
 content:
+  field_abstract:
```

Comment:
Likewise, the pdfjs view adds a modifies of fields like access rights, alternative title, etc etc.

Reply:
This is just an Entity Display View that shows what's available to display if the UI grabs this Display.  Doesn't touch the content type, merely a UI thing. 

This change slipped in and I will take it out (it's a bit innocuous, since those pages aren't styled yet).